test/src/worker/testplane.js

```diff
@@ -77,50 +77,51 @@ describe("worker/hermione", () => {
 
         describe("loading of plugins", () => {
             it("should load plugins", () => {
-                Hermione.create();
+                Testplane.create();
 
                 assert.calledOnce(pluginsLoader.load);
             });
 
-            it("should load plugins for hermione instance", () => {
-                Hermione.create();
+            it("should load plugins for testplane instance", () => {
+                Testplane.create();
 
-                assert.calledWith(pluginsLoader.load, sinon.match.instanceOf(Hermione));
+                assert.calledWith(pluginsLoader.load, sinon.match.instanceOf(Testplane));
             });
 
             it("should load plugins from config", () => {
                 Config.create.returns(makeConfigStub({ plugins: { "some-plugin": true } }));
 
-                Hermione.create();
+                Testplane.create();
 
                 assert.calledWith(pluginsLoader.load, sinon.match.any, { "some-plugin": true });
             });
 
-            it("should load plugins with appropriate prefix", () => {
-                Hermione.create();
+            // testplane does not support its own plugin prefixes.
+            it("should load plugins with deprecated hermione prefix", () => {
+                Testplane.create();
 
                 assert.calledWith(pluginsLoader.load, sinon.match.any, sinon.match.any, "hermione-");
             });
```

Comment:
Loading plugins with implicit "testplane-" or "@testplane/" would not be supported.